src/Sparse/Row.jl

```diff
@@ -537,13 +564,18 @@ function div(A::SRow{T}, b::Integer) where T
   return div(A, base_ring(A)(b))
 end
 
+@doc raw"""
+  divexact(A::SRow, b::NCRingElem) -> SRow
+
+Returns the left divexact $A/b$
```

Comment:
```suggestion
Returns the left divexact $b\A$
```
(I think???)

Reply:
no idea. maybe @thofma or @fingolfin know?

Reply:
I would follow what we say in other docstrings for `divexact` and friends and be explicit about the defining property of the return value:
```suggestion
Return a sparse row $C$ such that $a = b \cdot C$.
```
Or drop the "sparse row" (depending on doc strings for `SRow` do):
```suggestion
Return $C$ such that $a = b \cdot C$.
```